Hunting Queries/BehaviorAnalytics/Anomalous Data Access.yaml

```diff
@@ -0,0 +1,18 @@
+eid: 8cf3c78e-cd10-4bfb-bd69-d62dc7f375f1
+name: Anomalous Data Access
+description: |
+ 'Adversaries may access data objects from cloud storage.  The query below generates an output of all users performing a "read" operation regarding data or files, where one or more features of the activitiy deviates from the user, his peers or the tenant profile.'
+requiredDataConnectors:
+  - connectorId: BehaviorAnalytics
+    dataTypes:
+        - BehaviorAnalytics
+tactics:
+  - Collection
+relevantTechniques:
+  - T1530
+query: |
+    let operations = dynamic(['Export an existing database']);
+    BehaviorAnalytics
+    | where ActionType in(operations)
+    | where ActivityInsights contains "True"
```

Comment:
It is possible to use "has" here instead of "contains".  It is much more efficient.

Reply:
done